src/app/config/projectProperties.json

```diff
@@ -122,6 +122,19 @@
             "supportedHosts": [
                 "Excel"
             ]
+        },
+        "json-based-manifest": {
+            "displayname": "[Preview] Json Based Add-in Project",
+            "templates": {
+                "typescript": {
+                    "repository": "https://github.com/OfficeDev/Office-Addin-TaskPane",
```

Comment:
I think this has to point to the branch in the repo doesn't it?

Reply:
The problem is that the code already has some functions to use specific branches.
So I was modifying these functions instead of putting the branch in the URL, as if I just put the branch in the URL, it wouldn't be enough (as the code would be adding even more branch information).

Reply:
What parts of the code are looking at specific branches?

Reply:
OK . . . I think I get it now.  I do think you want the repository to be the json template branch, but the "branch" and "prerelease" should not be the standard branches used in other property sets.  If you look at the CF template it's has a separate set of branches for the code and for the yo office and for the prerelease.

Reply:
Yeah, I understand what you mean. I will try to change those values and see if I can get it to work